app/javascript/mastodon/locales/zh-CN.json

```diff
@@ -49,9 +49,9 @@
   "compose_form.placeholder": "在想啥？",
   "compose_form.publish": "嘟嘟",
   "compose_form.publish_loud": "{publish}！",
-  "compose_form.sensitive": "将媒体文件标记为“敏感内容”",
-  "compose_form.spoiler": "将部分文字隐藏于警告消息之后",
-  "compose_form.spoiler_placeholder": "隐藏文字的警告消息",
+  "compose_form.sensitive": "将媒体文件标记为敏感内容",
+  "compose_form.spoiler": "折叠嘟文内容",
+  "compose_form.spoiler_placeholder": "请填写折叠部分的警告消息",
```

Comment:
I don't think it helps to go from a description to an instruction here.

Reply:
@Artoria2e5 I'll change it back